### ISSUE TITLE
Field value documentation

### ISSUE THREAD

**Issue by LorenzoSapora** [timothyasp/nova-badge-field#10]
Hey Tim,

I can't seem to replace the displayed value, with a label

Here's a snippet:

```
Badge::make('Tier', 'tier')
->options(['1' => 'T1', '2' => 'T2'])
->colors([
   "1" => [
      "background" => '#333,
      "color" => '#fff',
   ],
   "2" => [
      "background" => '#fff',
      "color" => '#333',
   ]
]),
```

I, for the life of me, can't seem to get T1/T2 to display over the colors key.

I'll write up a block for your README once we get this cleared up ;)

**Comment by timothyasp** [comment]
Did you try adding `->displayUsingLabels()` ?

This Badge field is just a wrapper around a `Select` field https://nova.laravel.com/docs/1.0/resources/fields.html#select-field

**Comment by LorenzoSapora** [comment]
I did try that, and nothing changed. My next step was to attempt to wrap options in arrays:

```
options(['1' => ['label' => 'T1']])
->colors( same as above )
->displayUsingLabels()
```

I then wrote out a full select::make() with all the additions, then changed it to badge, and it no longer worked. Quite odd. 

I even attempted extending select directly with your modifications, but ended up with the same result.

I could be doing something wrong, but I’m at a loss as to what.

**Comment by timothyasp** [comment]
Sounds like it's a bug with the `Badge` code then, i'll take a look

**Comment by Chema-TM** [comment]
I try to show field displaying Labels, but nothing happens. @timothyasp please, can you add support to that feature? It will be amazing to allow translatable badges

**Comment by lazychaser** [comment]
@Chema-TM you need to specify colors for Labels, not Values (this is confusing for me, but it is how it is)

**Comment by LorenzoSapora** [comment]
@lazychaser Yes, but that's not what this issue is about. Functionality from the parent class is broken when using this module.

**Comment by IAntonioDev** [comment]
> @Chema-TM you need to specify colors for Labels, not Values (this is confusing for me, but it is how it is)

This worked for me!

**Comment by timcv** [comment]
@timothyasp i think #13 fixes this.

**Comment by SamBellerose** [comment]
Can someone provide a complete example that works? I'm still trying to figure out how to change my Labels name and colors. Thanks!

**Comment by timothyasp** [comment]
This was fixed by @timcv in #13.   You can now use `->displayUsingLabels()` .  Tagged and pushed as `1.0.3`
